ideasbox/serveradmin/tests/conftest.py

```diff
@@ -3,6 +3,13 @@
 from ..backup import Backup
 
 DATA_ROOT = 'ideasbox/serveradmin/tests/data'
+try:
+    from io import StringIO
+except ImportError:  # Python < 3
+    from StringIO import StringIO
+from mock import patch
```

Comment:
Wrong import order, use [isort](https://github.com/timothycrosley/isort) to help you follow the conventions.